crates/amaru/src/bin/amaru/cmd/import.rs

```diff
@@ -286,14 +404,28 @@ pub async fn run(args: Args) -> miette::Result<()> {
 
     let epoch = epoch_from_slot(point.slot_or_default());
 
-    db.next_snapshot(epoch).into_diagnostic()?;
+    {
+        let _snapshot_span = info_span!(
+            target: EVENT_TARGET,
+            "snapshot"
+        )
+        .entered();
+        db.next_snapshot(epoch).into_diagnostic()?;
+    }
 
-    db.with_pools(|iterator| {
-        for (_, pool) in iterator {
-            pools::Row::tick(pool, epoch + 1)
-        }
-    })
-    .into_diagnostic()?;
+    {
+        let _pool_span = info_span!(
+            target: EVENT_TARGET,
+            "pools"
+        )
+        .entered();
+        db.with_pools(|iterator| {
+            for (_, pool) in iterator {
+                pools::Row::tick(pool, epoch + 1)
+            }
+        })
+        .into_diagnostic()?;
+    }
 
     Ok(())
 }
```

Comment:
I appreciate the effort in this file but I still want to cry a little because I greatly reworked that file in #42.

Reply:
Hah that's fine we can discard the changes; like I said, it was mostly an interesting test before for me to make sure tracing was working etc.

Reply:
I'd still keep the shutdown fix though

Reply:
As said, cherry-picked already 🫡